cernopendata/templates/cernopendata_records_ui/records/detail.html

```diff
@@ -338,29 +338,32 @@ <h4> Export </h4>
     }
   ]);
 
-
 recordApp.controller(
   'filePreviewModalInstanceCtrl', [
     '$scope',
     '$uibModalInstance',
     'content',
-  function ($scope, $uibModalInstance, content){
+    'clipboard',
+  function ($scope, $uibModalInstance, content, clipboard){
     var $ctrl = this;
 
+    $ctrl.msg = null;
     $scope.contents = content;
 
     $scope.codemirrorLoaded = function(_editor){
-      setTimeout(function(){
-        _editor.refresh()
-      });
+      $ctrl.copyEditor = function() {
+        clipboard.copyText(content);
```

Comment:
This seems to copy the full cmsDriver script etc.  Is it possible to copy only the parts user selectied? Use case: the script may be big and the person only wants to copy the highlighted part.

If it is too time consuming, we can merge this already :+1: